### ISSUE TITLE
Feature Request: Inline Notification Settings in Hub

### ISSUE THREAD

**Issue by bacoords** [WordPress/wp-feature-notifications#104]
## What problem does this address?
Based on a [comment](https://make.wordpress.org/core/2022/07/21/request-for-feedback-feature-notifications-proof-of-concept/#comment-43432) could we offer inline notification settings in the sidebar notification hub?

## What is your proposed solution?
> 3. I love the Hub, but it needs a close affordance/button. I would also love for:
a) Notification settings for a given source to be available in the hub. e.g. click a button on the notification in the hub and get the chance to disable/enable notifications, without needing to visit the separate settings page. This would enable me to say “I don’t want these notices” but while keeping me in the flow of viewing my notices. Like how the iOS notification center works (see [this](https://share.cleanshot.com/deG2KVCgvyvIQp69LF1r), [this](https://share.cleanshot.com/PhoXEFHbaZ2VuiwOUPGD) and [this](https://share.cleanshot.com/2HfEMXvnWPRMX3Rlwr2w))

**Comment by folletto** [comment]
I think this is a good idea, but I'd not consider it for v1. Or depending on the framing, more of a nice to have once all the fundamentals are in place.

**Comment by johnhooks** [comment]
I agree there should be an option to disable notifications from a source directly in the Hub, or at the very minimum something to give the user the chance to opt out of receiving notifications from a source on their first notification from that source. Repetitively diving into a menu to disable a new notification source, could easily frustrate users with having the feature of notifications in WordPress.

**Comment by folletto** [comment]
As noted above, I would def consider this option, but it's not a minor addition and I would schedule this for at least a v1.2 (after a v1.1 to fix urgent feedback from v1.0 ;) ).

Also note that notification sources won't magically appear. They are likely linked to a newly installed plugin, so there's at least one level of indirection there.

**Comment by johnhooks** [comment]
I agree a new menu or UI element complicates the design. But we are creating a notification system, and it could initially be as simple as a notification sent from WordPress, "We noticed a new notification subscription added to your profile, do you want continue receiving notifications from such and such source?"
